kubeflow/core/cert-manager.libsonnet

```diff
@@ -0,0 +1,181 @@
+        group: "certmanager.k8s.io",
+        version: "v1alpha1",
+        names: {
+          kind: "Certificate",
+          plural: "certificates",
+        },
+        scope: "Namespaced",
+      },
+    },
+
+    clusterIssuerCRD:: {
+      apiVersion: "apiextensions.k8s.io/v1beta1",
+      kind: "CustomResourceDefinition",
+      metadata: {
+        name: "clusterissuers.certmanager.k8s.io",
+      },
+        
+      spec: {
+        group: "certmanager.k8s.io",
+        version: "v1alpha1",
+        names: {
+          kind: "ClusterIssuer",
+          plural: "clusterissuers",
+        },
+        scope: "Cluster",
+      },
+    },
+
+    issuerCRD:: {
+      apiVersion: "apiextensions.k8s.io/v1beta1",
+      kind: "CustomResourceDefinition",
+      metadata: {
+        name: "issuers.certmanager.k8s.io"
+      },
+      spec: {
+        group: "certmanager.k8s.io",
+        version: "v1alpha1",
+        names: {
+          kind: "Issuer",
+          plural: "issuers",
+        },
+        scope: "Namespaced",
+      },
+    },
+    
+    serviceAccount:: {
+      apiVersion: "v1",
+      kind: "ServiceAccount",
+      metadata: {
+        name: "cert-manager",
+        namespace: namespace,
+      }
+    },
+
+    clusterRole:: {
+      apiVersion: "rbac.authorization.k8s.io/v1beta1",
+      kind: "ClusterRole",
+      metadata: {
+        name: "cert-manager",
+      },
+      rules: [
+        {
+          apiGroups: ["certmanager.k8s.io"],
+          resources: ["certificates", "issuers", "clusterissuers"],
+          verbs: ["*"],
+        },
+        {
+          apiGroups: [""],
+          resources: ["secrets", "events", "endpoints", "services", "pods"],
+          verbs: ["*"],
+        },
+        {
+          apiGroups: ["extensions"],
+          resources: ["ingresses"],
+          verbs: ["*"],
+        },
+      ],
+    },
+
+    clusterRoleBinding:: {
+      apiVersion: "rbac.authorization.k8s.io/v1beta1",
+      kind: "ClusterRoleBinding",
+      metadata: {
+        name: "cert-manager",
+      },
+      roleRef: {
+        apiGroup: "rbac.authorization.k8s.io",
+        kind: "ClusterRole",
+        name: "cert-manager",
+      },
+      subjects: [
+        {
+          name: "cert-manager",
+          namespace: namespace,
+          kind: "ServiceAccount",
+        },
+      ],
+    },
+
+    deploy:: {
+      apiVersion: "apps/v1beta1",
+      kind: "Deployment",
+      metadata: {
+        name: "cert-manager",
+        namespace: namespace,
+        labels: {
+          app: "cert-manager",
+        },
+      },
+      spec: {
+        replicas: 1,
+        template: {
+          metadata: {
+            labels: {
+              app: "cert-manager",
+            },
+          },
+          spec: {
+            serviceAccountName: "cert-manager",
+            containers: [
+              {
+                name: "cert-manager",
+                image: "quay.io/jetstack/cert-manager-controller:v0.2.3",
```

Comment:
nit: Can we make these images local variables at the top of the file so that they are easy to udpate?